eng/liveBuilds.targets

```diff
@@ -223,6 +223,8 @@
             ResolveLibrariesRuntimeFilesFromLocalBuild" />
 
   <PropertyGroup>
-    <BundledRuntimeIdentifierGraphFile>$(RuntimeIdGraphDefinitionFile)</BundledRuntimeIdentifierGraphFile>
+    <!-- Keep in sync with outputs defined in Microsoft.NETCore.Platforms.csproj. -->
```

Comment:
This needs to be in the `.targets` file otherwise `Microsoft.NET.Sdk` overwrites it.